### PR TITLE
Trafodion 2884

### PR DESCRIPTION
**Pull request by zcorrea**
Implements: https://issues.apache.org/jira/browse/TRAFODION-2884

### CODE REVIEW

File: core/sqf/sqenvcom.sh [hunk 1]
```diff
@@ -673,17 +673,31 @@ export SQ_LUNMGR_VERBOSITY=1
...
+#   NAME-SERVER - to disable process replication and enable the name-server
+#
+# Uncomment the next environment variable
+#export SQ_NAMESERVER_ENABLED=1
+if [[ "$SQ_NAMESERVER_ENABLED" == "1" ]]; then
+  export NS_COMM_PORT=23370
+  export NS_SYNC_PORT=23360
+  export NS_M2N_COMM_PORT=23350
+  export MON2MON_COMM_PORT=23340
+fi
```
Comment (svarnau) [inline core/sqf/sqenvcom.sh]: PORT numbers should not be hard-coded here. I think most default ports are coded elsewhere, but these need to at leaset be coded to allow over-ride, such as:
export NS_COMM_PORT=${NS_COMM_PORT:-23370}

Comment (zcorrea) [reply core/sqf/sqenvcom.sh]: Is the '-' part of the syntax?

Comment (svarnau) [reply core/sqf/sqenvcom.sh]: Yes

Comment (zcorrea) [reply core/sqf/sqenvcom.sh]: I'll fix this. Thanks!

**Comment by sureshsubbiah** [comment]
+1

**Pull request merged**